### DISCUSSION TITLE
avoid rendering menu when I try to run command `screenshot` / `screenshot window` by menu

### DISCUSSION THREAD

**Discussion by hooke007** [tomasklaen/uosc#333] [General]
Is there any ideas how to achieve this?

```
###input.conf
# screenshot window   #! screenshot
```

Current:
<img src="https://user-images.githubusercontent.com/41094733/196226006-dd8b3ad3-5959-4d14-b3d0-dbaef5993694.jpg" width=400>

My temp solution is using the external script to add the new command to take screenshots with latency.

**Comment by tomasklaen** [comment]
Yes that's the only workaround I can think of. If we wanted to support this we'd have to do the same thing: have a timeout for when the next frame with no menu is rendered and only then execute the command. 

But this is pretty much the only case this is needed. For every other command I'd rather have no latency than waiting for menu to close, so I don't see myself implementing it.

It'd also mean we can no longer fade the menu out, because it'd add too much latency between click and command execution.

Yeah this is just too many drawbacks to support a single command which you can just bind to a shortcut and have none of these issues.
